### PR TITLE
GEOPY-1212: address reports from static code analyzers [WIP]

### PR DESCRIPTION
**Pull request by sebhmg**

### CODE REVIEW

**Comment by sebhmg** [comment]
waiting for https://github.com/MiraGeoscience/geoapps-utils/pull/11 to be merged first

File: geoapps_utils/application/dash_application.py [hunk 1]
```diff
@@ -62,20 +66,30 @@ def __init__(
...
+        self.workspace = self.params.geoh5
         self.workspace.open()
         if self._driver_class is not None:
-            self.driver = self._driver_class(self.params)  # pylint: disable=E1102
+            self.driver = self._driver_class(self.params)
 
-        self.app = None
+    @property
+    @abstractmethod
+    def app(self) -> Dash:
```
Comment (sebhmg) [inline geoapps_utils/application/dash_application.py:91]: **todo:** derived classes in all repos need to implement this

File: pylintrc [hunk 2]
```diff
@@ -22,7 +22,7 @@ analyse-fallback-blocks=no
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
 extension-pkg-allow-list=numpy,
-                         scipy
+                         PySide2
```
Comment (sebhmg) [inline pylintrc:25]: this tells pylint to load the compiled libraries for PySide2 so that is can see the symbols

**Comment by domfournier** [comment]
Looks like changes were included in #21. Can we close this?

**Pull request closed without merging**